integration/examples_nodejs_test.go

```diff
@@ -82,6 +83,19 @@ func TestLogs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestMisc(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: filepath.Join(getCwd(t), "misc-services"),
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				repoName := stack.Outputs["repoName"].(string)
+				assert.Containsf(t, "testrepob5dda46f", repoName, "Expected repoName to contain 'testrepob5dda46f'; got %s", repoName)
```

Comment:
This is the CDK generated suffix so it is a hash based on the construct tree. It will be the same everytime.

Reply:
Ah and I guess it's fine anyways because there shouldn't be any change to the tree as those would essentially cause a replacement (because the name changes), right?
Thanks for the explanation!